gatsby-node.js

```diff
@@ -4,7 +4,7 @@ const webpack = require('webpack')
 
 const trimLeft = (s, charlist) => {
   if (charlist === undefined) {
-    charlist = '\s'
+    charlist = 's'
```

Comment:
We reverted this, now it's back. Please revert it again.

Reply:
Dude, please, it's still not correct. Just accept this patch, please:

```suggestion
    charlist = '\s'
```